src/data/i18n/locales/json/currentlySupportedLangs.jsx

```diff
@@ -0,0 +1,18 @@
+import "./ar.json"
+import 'react-intl/locale-data/ar';
+
+import "./es_419.json"
+import 'react-intl/locale-data/es';
+
+import "./fr_CA.json"  //fr.json is empty ATM, but we do have data for fr_CA.json
+import 'react-intl/locale-data/fr';
+
+import "./zh_CN.json"
+import 'react-intl/locale-data/zh';
+
+// Methodology: These languages (plus the default english) are the only ones that appear on edx.org
+// This was determined by inspecting the 'released_languages' method in the following file:
+//    https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/lang_pref/api.py
+//
+// Further, I hit the read replica in order to confirm the live values, which are currently:
+//    en, fr, zh-cn, es-419, ar
```

Comment:
And this is the other piece of that new entry point - a jsx file that does nothing other than import some things. I hand-crafted the list of specific files, can't seem to find a way to do "import all the possible languages here". If you have a better idea, LMK.

Reply:
Well, hopefully this list wouldn't change that often. I'm not sure how we will know to update this file if it changes though...

Reply:
Yeah, it's less-than ideal for sure. FWIW though, the dark lang setting in the read-replica seemed to be quite stable; the values there haven't changed very frequently.

I suppose we're going to rely on a combination of (1) developers (us) knowing that this limitation exists in SFE and (2) tickets being made to update language options when required.